integrations/optimizely/test/index.test.js

```diff
@@ -1051,37 +772,36 @@ describe('Optimizely', function() {
         window.optimizely.newMockData[2542102702].isActive = false;
         analytics.initialize();
         executeAsyncTest(done, function() {
-          analytics.didNotCall(analytics.track);
+          sinon.assert.notCalled(analytics.track);
         });
       });
     });
   });
 
-  describe('after loading', function() {
-    beforeEach(function(done) {
-      analytics.once('ready', done);
+  describe('#track', function() {
+    beforeEach(function() {
       analytics.initialize();
-      mockBothOptimizelyDataObjects();
-      analytics.page();
```

Comment:
I'm not sure if this was doing anything. There may be _some_ value in testing an `analytics.page` call without the optional name argument, but I suspect this is not the correct place for that.